modules/adhashBidAdapter.js

```diff
@@ -0,0 +1,90 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+
+export const spec = {
+  code: 'adhash',
+  url: 'https://bidder.adhash.org/rtb?version=1.0&prebid=true',
+  supportedMediaTypes: [ BANNER ],
+
+  isBidRequestValid: (bid) => {
+    try {
+      const { publisherId, platformURL } = bid.params;
+      return (
+        typeof publisherId === 'string' &&
+        publisherId.length === 42 &&
+        typeof platformURL === 'string' &&
```

Comment:
Might be useful to add a check for 'adhash.org' to exist in the url?

Reply:
The URL will not always include adhash.org as every client has the option to install the platform on their own URL.